src/aws-s3/orb.yml

```diff
@@ -102,7 +102,7 @@ commands:
           aws-secret-access-key: << parameters.aws-secret-access-key >>
           aws-region: << parameters.aws-region >>
       - deploy:
-          name: Deploy to S3
+          name: S3 Sync << parameters.from >> -> << parameters.to >>
```

Comment:
how about just `name: S3 Sync`  instead ?

Reply:
It is easier to understand and better than `Deploy to S3` 😄

Reply:
Remove `<< parameters.from >> -> << parameters.to >>`.
Please confirm 🙏